buildout.cfg

```diff
@@ -30,7 +30,7 @@ bleach = 2.1.2
 certifi = 2018.1.18
 cffi = 1.13.2
 collective.recipe.sphinxbuilder = 1.0
-colorama = 0.3.9
+colorama = 0.4.3
```

Comment:
@mauritsvanrees I needed to update this to get the Docker image to build for tests.